src/main.cpp

```diff
@@ -39,6 +39,7 @@ int main(int argc, char *argv[]) {
   int batch;
   int threads;
   bool debug;
+  bool openslideShift;
```

Comment:
Maybe call this dropFirstRowAndColumn to make it more clear?

Reply:
done